obspy/io/nordic/ellipse.py

```diff
@@ -0,0 +1,329 @@
+"""
+Routines for error ellipses
+ - Calculate an ellipse from covariance matrix
+ - See if a point is inside an ellipse
+ - Calculate the angle subtended by an ellipse (for back-azimuth uncert)
+ - Plot an ellipse
+
+Still missing:
+ - ellipsoids (3D ellipses)
+ - to_uncert()  (convert ellipse to covariance matrice, needed to
+                 convert QuakeML to nordic, for example)
+"""
+
+import numpy as np
+import matplotlib.pyplot as plt
+import copy
+
+eps = np.finfo(float).eps
+
+
+class _ellipse:
```

Comment:
I think there are a few syntax issues in here - might be worth running flake8 over this locally.

Reply:
flake8 seemed happy with ellipse.py (after the eps removal at least).
core.py and utils.py were another story after my additions...
I fixed them all